### ISSUE TITLE
Problems installing gecco via conda

### ISSUE THREAD

**Issue by tamuanand** [zellerlab/GECCO#3]
Hi

I installed `gecco` via `conda install -c bioconda -y gecco`

When I try calling `gecco` within my conda activated environment,  I get this error message
```
Traceback (most recent call last):
  File "<path_to>/envs/gecco/lib/python3.6/site-packages/gecco/cli/commands/_main.py", line 22, in <module>
    import importlib.metadata as importlib_metadata
ModuleNotFoundError: No module named 'importlib.metadata'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<path_to>/envs/gecco/bin/gecco", line 10, in <module>
    sys.exit(main())
  File "<path_to>/envs/gecco/lib/python3.6/site-packages/gecco/cli/__init__.py", line 9, in main
    from .commands._main import Main
  File "<path_to>/envs/gecco/lib/python3.6/site-packages/gecco/cli/commands/_main.py", line 24, in <module>
    import importlib_metadata
ModuleNotFoundError: No module named 'importlib_metadata'
```

Thanks in advance

**Comment by tamuanand** [comment]
Hi

I have some more updates on this issue and `fixes`

Based on the original issue, I first tried installing `importlib-metadata` via conda-forge. Doing so did not give me errors when calling `gecco`. 

However upon running `gecco`, it now started complaining about `importlib.resources`
```
ModuleNotFoundError: No module named 'importlib.resources'
```

After installing `importlib_resources` , gecco now runs fine

Here is a step by step installation - if it helps anybody.  I also list below a faster method using [conda-lock](https://github.com/conda-incubator/conda-lock) 
```
conda create -n gecco python=3.7
conda activate gecco
conda install -c bioconda -y gecco
conda install -c conda-forge importlib-metadata
conda install -c conda-forge importlib_resources
```

**_Suggested faster method_**

1. Download conda-lock (via conda or pip)
2. Create `environment.yml` file as below
3. Use conda-lock to create a lock file for the above `environment.yml` file
4. Use the lock file to create the new environment

```
# You can use this file to create a conda environment 
#  conda env create -f environment.yml
# or use conda-lock via the locked file
name: gecco
channels:
  - bioconda
  - conda-forge
  - defaults
dependencies:
  - gecco
  - importlib-metadata
  - importlib_resources
 ```

**_Details on the suggested faster method via conda-lock_**

```
# You will do all this below from your conda base environment

conda-lock   -f environment.yml  -p linux-64 

# The above will create a file called conda-linux-64.lock

conda create --name gecco --file conda-linux-64.lock

# The conda environment will get created and you can use it
# conda activate gecco
```

**Comment by althonos** [comment]
HI @tamuanand,

thanks for the detailed report ! `importlib.metadata` and `importlib.resources` are modules that were added in Python 3.7 (resp. 3.8) but for which backports exist. The `conda` recipe in the `bioconda` channel, which I created, should normally install these whenever your Python version is too low; I'll have a look with the Bioconda maintainers since from what you're writing here it obviously doesn't work.
